contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -437,6 +454,7 @@ parameters:
         filter:
           key: name
           value: dest_service_count
+        filters: []
```

Comment:
`filter` is deprecated in favor of `filters` so we should convert these to arrays